test/jdk/java/io/File/libGetXSpace.c

```diff
@@ -77,17 +77,19 @@ Java_GetXSpace_getSpace0
         }
     }
 
-    LPCWSTR path = (LPCWSTR)chars;
+    LPCWSTR path = (LPCWSTR)strchars;
 
     if (pfnGetDiskSpaceInformation != NULL) {
         // use GetDiskSpaceInformationW
         DISK_SPACE_INFORMATION diskSpaceInfo;
         BOOL hres = pfnGetDiskSpaceInformation(path, &diskSpaceInfo);
```

Comment:
The strchars/path could be released before checking the return; keeping the cleanup in the common flow.
Here and below after calling GetDiskFreeSpaceExW.
Your call.

Reply:
Good idea; will update.

Reply:
So changed in 0ecf89b1d060bb8c1a35402ec108cbe9f3f35bb9.